src/components/internal/ImageUploader.component.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { ChangeEvent, Component, Fragment } from 'react';
+import styles from './ImageUploader.component.module.scss';
+import {Progress} from '@lib/fileUpload/progress';
+import { FileUploadSessionFactory } from '@lib/fileUpload/fileUploadSession';
+
+interface Props {
+    fileUploadSessionFactory: FileUploadSessionFactory;
+    uploadLabel: string;
+    onFileUploadInit: () => Promise<number>;
+    onFileUploadFinished: (fileUploadSessionId: number) => Promise<void>;
```

Comment:
```suggestion
    onFileUploadFinished?: (fileUploadSessionId: number) => Promise<void>;
```

Let's make this callback optional. The parent of this callback may decide they are not interested in doing anything when this happens and hence doesn't pass in anything.